README.md

```diff
@@ -212,6 +212,22 @@ Also, note that this will [not work](https://github.com/sharkdp/bat/issues/1145)
 
 The [`prettybat`](https://github.com/eth-p/bat-extras/blob/master/doc/prettybat.md) script is a wrapper that will format code and print it with `bat`.
 
+#### Highlighting `--help` messages
+
+You can use bat to colorize help text: `$ cp --help | bat -plhelp`
```

Comment:
In other places in the readme, we put `bat` in backticks.
```suggestion
You can use `bat` to colorize help text: `$ cp --help | bat -plhelp`
```

Reply:
My bad! Fixed.

I also included a line to (try to) channel issues my way.